RdlMigration/ConvertRDL.cs

```diff
@@ -21,6 +22,11 @@ namespace RdlMigration
     /// </summary>
     public class ConvertRDL
```

Comment:
you need to add tests for the new logic added

Reply:
:(

Reply:
added!